src/pages/user/components/infoUser/detailUser/detailUser.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Col, Container, Row } from 'reactstrap';
+import { AUTHORS, TAB_INFO_USER } from '../../../../../constants';
+import { theme } from '../../../../../styles/theme';
+import * as F from '../styled-infoUSer'
+import * as D from '../../../../movies/components/detailMovie/style-detailMovie'
+
+function DetailUser({ tab }) {
+    return (
+        <Container style={{ display: tab === TAB_INFO_USER ? 'block' : 'none' }}>
+            <F.TabItem>
+                <Row>
+                    <Col lg={6} >
+                        <img src="https://naht-cinema-api.herokuapp.com/src/resoures/defaulavartar.png" alt="avarta" />
```

Comment:
```suggestion
                    <Col lg={6}>
                        <img src="https://naht-cinema-api.herokuapp.com/src/resoures/defaulavartar.png" alt="avarta" />
```
Download ảnh về rồi import vào thay vì để cả link như này nhé
Có thể cho ảnh này là defaultUserAvatar luôn

Reply:
cái này ở pull tiếp theo em có để như anh nói vậy ạ, anh có thể merge giúp em rồi em đây pull kia lên ạ